src/styles/utils.scss/_mixins.scss

```
@@ -0,0 +1,4 @@
+@mixin square-size {
```

```suggestion
@mixin square-size ($size) {
```
Better get value as argument